### PR TITLE
Show user count in users page

### PR DESCRIPTION
**Pull request by jpuccinit**
#RAD-1090

### CODE REVIEW

**Comment by terrypacker** [comment]
@jpuccinit looks good to me, @jazdw you want to take a look at this?  If not I am fine to merge it.

File: UI/web-src/ngMango/components/userTable/userTable.html [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-                            <md-menu-content width="3" class="ma-user-table" md-colors="::{background: 'background-hue-2'}">
+                            <md-menu-content width="3" class="ma-user-table"
+                                md-colors="::{background: 'background-hue-2'}">
                                 <md-menu-item ng-if-start="$ctrl.selectMultiple">
                                     <md-button ng-disabled="true">
-                                        <span ma-tr="ui.app.xItemsSelectedLong" ma-tr-args="[$ctrl.selectedItems.size]"></span>
+                                        <span ma-tr="ui.app.xItemsSelectedLong"
+                                            ma-tr-args="[$ctrl.selectedItems.size]"></span>
+                                        <span>&nbsp;</span>
+                                        <span ma-tr="ui.app.outOf" ma-tr-args="[$ctrl.pages.$total]"></span>
```
Comment (jazdw) [inline UI/web-src/ngMango/components/userTable/userTable.html:21]: Please avoid using translations for incomplete sentences. You should add a new translation for the complete sentence and use a single span.

File: UI/web-src/ngMango/components/userTable/userTable.html [hunk 2]
```diff
@@ -50,24 +56,30 @@
...
+                                            md-colors="::{background: 'background-hue-2'}">
                                             <md-menu-item ng-repeat="column in $ctrl.columns track by column.name">
                                                 <md-button ng-init="model = $ctrl.getSelectedColumnsModel(column)"
-                                                        ng-click="model.value = !model.value; $ctrl.selectedColumnsChanged()" md-prevent-menu-close
+                                                    ng-click="model.value = !model.value; $ctrl.selectedColumnsChanged()"
+                                                    md-prevent-menu-close
+                                                    ng-disabled="model.value && $ctrl.selectedColumns.length === 1">
+                                                    <md-checkbox ng-click="$event.stopPropagation()"
+                                                        md-prevent-menu-close md-no-ink="true" ng-model="model.value"
+                                                        ng-change="$ctrl.selectedColumnsChanged()"
```
Comment (jazdw) [inline UI/web-src/ngMango/components/userTable/userTable.html:81]: Please avoid reformatting lines that have not been modified. This makes it really hard for me to look at and tell what you have actually changed.

**Review by jazdw — CHANGES_REQUESTED** [review]

**Comment by jazdw** [comment]
@jpuccinit Please include a screenshot

**Comment by jpuccinit** [comment]
Screen Shot 2021-10-04 at 5 32 41 PM
Screen Shot 2021-10-04 at 5 32 54 PM

**Comment by jazdw** [comment]
@jpuccinit thank you. However there are still a lot of whitespace changes which I do not want to merge, please revert the whitespace changes and squash all of your changes into one commit.

**Review by jazdw — CHANGES_REQUESTED** [review]
@jpuccinit thank you. However there are still a lot of whitespace changes which I do not want to merge, please revert the whitespace changes and squash all of your changes into one commit.

File: UI/resources/i18n.properties [hunk 3]
```diff
@@ -823,7 +823,7 @@ ui.app.pointValueImport.always=Always
 ui.app.pointValueImport.result={0} value queued, {1} value skipped
 ui.app.clearTranslationsCache=Clear translations cache and reload
 ui.app.showUnacknowlegedEventsIcon=Show unacknowledged events icon in toolbar
-
+ui.app.xItemsSelectedLongOutOf={0, plural, =0{No items} =1{1 item} other{# items}} selected out of {1}
```
Comment (jazdw) [inline UI/resources/i18n.properties:826]: Add parenthesis around "out of". e.g. `1 item selected (out of 3)`

**Comment by jpuccinit** [comment]
@jazdw  I thing i fix  the whietespace problem

**Review by jazdw — APPROVED** [review]

**Pull request merged**